docs/get_started.md

```
@@ -26,70 +27,69 @@
-1. Ensure you have:
-   * Ubuntu 24.04 (recommended)
-   * x86_64 CPU
-   * Python 3.x
-   * Git
+ 1. Ensure you have:
 
    See [Support Matrix](support_matrix.md) for more information.
 
-2. **If you plan to use vLLM or SGLang**, you must also install:
+ 2. **If you plan to use vLLM or SGLang**, you must also install:
       * etcd
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)